modules/react/button/lib/Button.tsx

```diff
@@ -56,10 +56,7 @@ export const Button = createComponent('button')({
         )}
         {children && <BaseButton.Label>{children}</BaseButton.Label>}
 
-        {icon && baseIconPosition === 'only' && (
-          <BaseButton.Icon size={size} icon={icon} shouldMirrorIcon={shouldMirrorIcon} />
-        )}
-        {icon && baseIconPosition === 'end' && (
+        {icon && baseIconPosition && ['only', 'end'].includes(baseIconPosition) && (
```

Comment:
`only` isn't technically a valid prop right? I forgot why we need `only`.

Reply:
Correct.

We do need it for the modifiers though since`only` and no icon have different padding.